prboom2/src/st_stuff.c

```diff
@@ -267,9 +267,6 @@ static int veryfirsttime = 1;
 // used for timing
 static unsigned int st_clock;
 
-// used for making messages go away
-static int st_msgcounter=0;
```

Comment:
I assume this was superseded by your hud overhaul

Reply:
Seems like it, that one can definitely go.